README.md

```diff
@@ -162,7 +162,7 @@ The Canopy-App input data in [Table 2](#table-2-canopy-app-required-input-variab
 | `ch`                             | Canopy height (m)                    | Globally extended GEDI data. Data Period=2020. Data frequency=Annual. ([Lang et al., 2023](https://doi.org/10.1038/s41559-023-02206-6)) |
 | `clu`                            | Canopy clumping index (dimensionless)       | GriddingMachine/MODIS. Data Period=2001-2017 Climatology. Data frequency=Monthly. ([Wei et al., 2019](https://doi.org/10.1016/j.rse.2019.111296)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |
 | `lai`                            | Leaf area index (m2/m2)                     | VIIRS-NPP. Data Period=2020. Data frequency=Daily, interpolated from original 8-day product. ([Myneni 2018](https://doi.org/10.5067/VIIRS/VNP15A2H.001)). Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |
-| `canfrac`                          | Canopy fraction (dimensionless)             | Based on [MODIS VCF](https://doi.org/10.5067/MODIS/MOD44B.061). Data Period=2020. Data frequency=Annual. Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |
+| `canfrac`                          | Canopy green vegetation fraction (dimensionless)             | Based on [VIIRS GVF]([https://doi.org/10.5067/MODIS/MOD44B.061](https://www.star.nesdis.noaa.gov/jpss/gvf.php)). Data Period=2020. Data frequency=Monthly. Extended globally for high latitudes using methods described [here](https://gmuedu-my.sharepoint.com/:w:/g/personal/whung_gmu_edu/EdglXmW2kzBDtDj1xV0alGcB1Yo2I8hzdyWGVGB2YOTfgw). |
```

Comment:
@angehung5  Is the VIIRS GVF still have the same link/reference as VCF?  Seems like this should be updated for GVF product reference.

Also, the link to your global extension and processing document is outdated, and should be changed to your updated [ARL tech memo](https://docs.google.com/document/d/1zzE1XkHRChuSw5J1a5vtb86e2850nNjg/edit?usp=sharing&ouid=105019685237374277307&rtpof=true&sd=true  ).    However this is your NOAA Google doc and should be made public in a sharepoint document.

Reply:
Sorry my bad, the link is updated now. 
The document is not necessarily outdated. A short description of GVF is added. I think it should be enough for now, will update it with the ARL tech memo once it gets published.